File: api/app/models/trash.rb
class Trash < ApplicationRecord
  belongs_to :region, foreign_key: 'region_id', optional: true, primary_key: 'name'

  enum name: {
    clothes: 0,
    plastic: 1,
    burnable: 2,
    bottles_and_oil: 3,
    unburnable: 4,
    paper: 5,
    nothing: 6
  }
end

File: front/domain/entity/Trash.ts
export enum TrashName {
	Clothes = 'clothes',
	Plastic = 'plastic',
	Burnable = 'burnable',
	BottolesAndOil = 'bottles_and_oil',
	Unburnable= 'unburnable',
	Paper = 'paper',
	Nothing = 'nothing',
}

export class Trash {
	constructor(
    public id: number | undefined,
    public name: TrashName,
    public regionId: number,
    public date: Date,
  ) {}
}
